warehouse/query-core/src/main/java/datawave/query/tables/ScannerFactory.java

```diff
@@ -563,18 +580,28 @@ protected void applyConfigs(ScannerSession scannerSession, String hintKey, Strin
         SessionOptions options = scannerSession.getOptions();
 
         if (consistencyLevelMap != null && !consistencyLevelMap.isEmpty()) {
-            String key = consistencyLevelMap.containsKey(hintKey) ? hintKey : tableName;
+            ScannerBase.ConsistencyLevel level = consistencyLevelMap.get(hintKey);
+            if (level == null) {
+                level = consistencyLevelMap.get(tableName);
+            }
 
-            if (consistencyLevelMap.containsKey(key)) {
-                options.setConsistencyLevel(consistencyLevelMap.get(key));
+            if (level == null) {
+                log.trace("no consistency level found for table: {} key: {}", tableName, hintKey);
+            } else {
+                options.setConsistencyLevel(level);
             }
         }
 
         if (executionHintMap != null && !executionHintMap.isEmpty()) {
-            String key = executionHintMap.containsKey(hintKey) ? hintKey : tableName;
+            Map<String,String> hint = executionHintMap.get(hintKey);
+            if (hint == null) {
+                hint = executionHintMap.get(tableName);
+            }
 
-            if (executionHintMap.containsKey(key)) {
-                options.setExecutionHints(executionHintMap.get(key));
+            if (hint == null) {
+                log.trace("no execution hint found for table: {} key: {} ", tableName, hintKey);
+            } else {
+                options.setExecutionHints(hint);
```

Comment:
this looks like the same code as the block around ~565, please use a method.

Reply:
good call -- commit forthcoming

Reply:
Commit can be found  here: https://github.com/NationalSecurityAgency/datawave/pull/2639